examples/spark/README.md

```diff
@@ -57,60 +57,89 @@ instructions for your platform.
-15/06/26 14:01:51 INFO SecurityManager: Changing modify acls to: root
-15/06/26 14:01:51 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: Set(root); users with modify permissions: Set(root)
-15/06/26 14:01:51 INFO Slf4jLogger: Slf4jLogger started
-15/06/26 14:01:51 INFO Remoting: Starting remoting
-15/06/26 14:01:52 INFO Remoting: Remoting started; listening on addresses :[akka.tcp://sparkMaster@spark-master:7077]
-15/06/26 14:01:52 INFO Utils: Successfully started service 'sparkMaster' on port 7077.
-15/06/26 14:01:52 INFO Utils: Successfully started service on port 6066.
-15/06/26 14:01:52 INFO StandaloneRestServer: Started REST server for submitting applications on port 6066
-15/06/26 14:01:52 INFO Master: Starting Spark master at spark://spark-master:7077
-15/06/26 14:01:52 INFO Master: Running Spark version 1.4.0
-15/06/26 14:01:52 INFO Utils: Successfully started service 'MasterUI' on port 8080.
-15/06/26 14:01:52 INFO MasterWebUI: Started MasterWebUI at http://10.244.2.34:8080
-15/06/26 14:01:53 INFO Master: I have been elected leader! New state: ALIVE
+$ kubectl logs spark-master-controller-5u0q5
+starting org.apache.spark.deploy.master.Master, logging to /opt/spark-1.5.1-bin-hadoop2.6/sbin/../logs/spark--org.apache.spark.deploy.master.Master-1-spark-master-controller-5u0q5.out
+15/10/26 18:00:35 INFO Remoting: Starting remoting
+15/10/26 18:00:36 INFO Remoting: Remoting started; listening on addresses :[akka.tcp://sparkMaster@spark-master:7077]
+15/10/26 18:00:36 INFO Utils: Successfully started service 'sparkMaster' on port 7077.
+15/10/26 18:00:36 INFO Master: Starting Spark master at spark://spark-master:7077
+15/10/26 18:00:36 INFO Master: Running Spark version 1.5.1
+15/10/26 18:00:36 INFO Utils: Successfully started service 'MasterUI' on port 8080.
+15/10/26 18:00:36 INFO MasterWebUI: Started MasterWebUI at http://spark-master:8080
+15/10/26 18:00:36 INFO Utils: Successfully started service on port 6066.
+15/10/26 18:00:36 INFO StandaloneRestServer: Started REST server for submitting applications on port 6066
+15/10/26 18:00:36 INFO Master: I have been elected leader! New state: ALIVE
+```
+
+If you created the Spark WebUI and waited sufficient time for the load balancer
+to be create, the `spark-webui` service should look something like this:
+
+```console
+$ kubectl describe services/spark-webui
+Name:                   spark-webui
+Namespace:              default
+Labels:                 <none>
+Selector:               component=spark-master
+Type:                   LoadBalancer
+IP:                     10.0.152.249
+LoadBalancer Ingress:   104.197.147.190
+Port:                   <unnamed>       8080/TCP
+NodePort:               <unnamed>       31141/TCP
+Endpoints:              10.244.1.12:8080
+Session Affinity:       None
+Events: [...]
 ```
 
+You should now be able to visit `http://104.197.147.190:8080` and see the Spark
```

Comment:
I'll add a note. I guess we should throw an nginx proxy in front of this but it makes the example just that much more obnoxious. :/